geo/parse.go

```diff
@@ -85,9 +85,11 @@ func ExtractGeoPoint(thing interface{}) (lon, lat float64, success bool) {
 			}
 		} else {
 			// geohash
-			lat, lon = DecodeGeoHash(geoStr)
-			foundLat = true
-			foundLon = true
+			if len(geoStr) <= 12 {
```

Comment:
Think it'd do good to declare a const GEO_HASH_MAX_LENGTH = 12 and use that here.

Reply:
yep, let me replace with geoHashMaxLength.